pkg/cmd/configs_tokens.go

```diff
@@ -173,6 +183,7 @@ func init() {
 	configsTokensCreateCmd.Flags().StringP("config", "c", "", "config (e.g. dev)")
 	configsTokensCreateCmd.Flags().Bool("plain", false, "print only the token, without formatting")
 	configsTokensCreateCmd.Flags().Bool("copy", false, "copy the token to your clipboard")
+	configsTokensCreateCmd.Flags().Duration("max-age", 0, "token will expire after specified duration (optional)")
```

Comment:
supernit: Should we maybe include an example duration? It took me a minute to figure out how to input this.

Reply:
Added a couple examples to the command's docs